.devauth/.gitignore

```diff
@@ -0,0 +1 @@
+microsoft_accounts.json
```

Comment:
Missing newline

Reply:
Actually, is this correct or did you intend to modify the real `.gitignore` file?

Reply:
TIL: Apparently you can have multiple .gitignore files. I strongly recommend against this, though. Please add a `.devauth/microsoft_accounnds.json` rule to the top-level .gitignore file instead.